config/initializers/okcomputer.rb

```diff
@@ -10,9 +10,9 @@
 OkComputer::Registry.register 'feature-resque-down', OkComputer::ResqueDownCheck.new
 
 # check for backed up resque queues: this is a low volume app, so the threshold is low
-Resque.queues.each do |queue|
-  OkComputer::Registry.register "feature-#{queue}-queue-depth", OkComputer::ResqueBackedUpCheck.new(queue, 5)
-end
+# Resque.queues.each do |queue|
```

Comment:
why is this being removed?  is it useless?

Reply:
This was just commented out for testing (because it hits redis). I've restore this.